### PR TITLE
chore: add discussion templates

### PR DESCRIPTION
**Pull request by ObserverOfTime**
The text is subject to change.

### CODE REVIEW

**Comment by clason** [comment]
I would remove the passive-aggressive checkmarks in favor of a simple text that is shown at the top of the issue template.

**Comment by ObserverOfTime** [comment]
Not the transfer ones though.

**Comment by clason** [comment]
The transfer ones, too. This is not a legally binding agreement, so we shouldn't pretend it is. Requesting transfer is granting permission enough.

File: .github/DISCUSSION_TEMPLATE/transfer.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+title: "Transfer "
+labels: [question]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        It's not possible to transfer a parser to an organization that you're not an admin of,
+        and we will not hand admin rights to everyone. Instead, you must temporarily transfer the
+        repository to one of the [maintainers](https://github.com/orgs/tree-sitter-grammars/people).
```
Comment (amaanq) [inline .github/DISCUSSION_TEMPLATE/transfer.yml:9]: would be nice for every org member to set their visibility to public to make this list more accurate/useful

Comment (ObserverOfTime) [reply .github/DISCUSSION_TEMPLATE/transfer.yml:9]: (Or set it to private if they don't want to handle transfer requests)

**Comment by ObserverOfTime** [comment]
Is this good to go?

**Comment by amaanq** [comment]
yeah

**Review by amaanq — APPROVED** [review]

**Pull request merged**